### PR TITLE
T-26 Antimaterial Sniper Rifle added to smartgunner; also, T-25 Smart assault rifle name change

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

The T-26 Antimaterial Sniper Rifle found a home in the smartgunner family. It has gun_skill_category = GUN_SKILL_SMARTGUN, so only smartgunner can use it.

T-26 is the most expensive out of all the smartgunner weapon so far. It spawns with 90 total rounds, so you have to make your shot count.

Since we have two rifles in smartgunner vendor, give the adjective assault to T-25 Smart assault rifle. Yeah, it's an assault rifle.

Bravemole approves. MJP, better luck next time.

## Why It's Good For The Game

This weapon belongs to marines, and marines barely use it. It was collecting web and dust in req, so it is time to give this baby a go.

Bravemole approves of putting this in smartgunner.

## Changelog
:cl:
add: T-26 Antimaterial Sniper Rifle added to smartgunner
expansion: T-25 smart rifle is now called T-25 smart assault rifle
balance: Smartgunners have new weapon!
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
image

**Comment by MereWalker** [comment]
Is this T26 the T81?

**Comment by Pariah919** [comment]
```
holy balls
why are you giving a 75 point weapon to sg

**Comment by Pariah919** [comment]
Stop trying to shove another weapon into your SG sniper fantasy and wait for someone to make either a new sprite or go grab an old one, because it needs specialized balance instead of shoving something like this in, which will probably break balance.

**Comment by novaepee** [comment]
> Is this T26 the T81? Or is this the Big anti material rifle.

Big anti-material. T-26

**Comment by novaepee** [comment]
> Stop trying to shove another weapon into your SG sniper fantasy and wait for someone to make either a new sprite or go grab an old sprite or something, because it needs specialized balance instead of shoving something like this in, which will probably break balance.

Someone else asked me to code it, and somehow Bravemole approves.

**Comment by Pariah919** [comment]
>Someone else asked me to code it, and somehow Bravemole approves.
>
Bravemole approving doesn't fix the issue that you're shoving a weapon that is a power weapon into smartgunners hands for no real reason than "People don't use it very often".

I do PR's for others all the time, it doesn't block you from criticism.

The main issue the AMR has which makes people not use it is twofold-

-I overbuffed the autosniper and never got feedback until recently, but people nerfed it recently to be better, though I think it needs to be moved to .75.
-The lase is broken and doesn't work half the time.

Giving it to SG is absurd because it let's you stack 6 AMRs and kill people instantly, and if you rework it entirely you should just make an entirely different gun.

**Comment by KrabSpider** [comment]
> > Stop trying to shove another weapon into your SG sniper fantasy and wait for someone to make either a new sprite or go grab an old sprite or something, because it needs specialized balance instead of shoving something like this in, which will probably break balance.
> 
> Someone else asked me to code it, and somehow Bravemole approves.

I never really asked. 😐

**Comment by SrSC0ndition** [comment]
Why did you restrict niche req weapon even more, putting it solely into hands of SG? Thanks for another taken opportunity for no reason lol

**Comment by Mrrpip** [comment]
But can't you stick a scope on the t-29 anyhow?

**Review by BraveMole — CHANGES_REQUESTED** [review]
I already said this need specialised balance, MJP is 100% right

**Comment by novaepee** [comment]
> I already said this need specialised balance, MJP is 100% right

Waiting for L-Catcher to give me balance values.

**Comment by KrabSpider** [comment]
> > I already said this need specialised balance, MJP is 100% right
> 
> Waiting for L-Catcher to give me balance values.

😐 
Just close the PR and buff the fire delay to 2 from 2.5. 

It doesnt need to be made roundstart, and I quite honestly played with the idea myself because I was seeking approval from Brave. 

If you're still going through with this; 
Decrease the mags given to 4. 
Decrease the sunder.

File: code/modules/projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -24,8 +24,8 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
 //Pow! Headshot
 
 /obj/item/weapon/gun/rifle/sniper/antimaterial
-	name = "\improper T-26 scoped rifle"
-	desc = "The T-26 is an IFF capable sniper rifle which is mostly used by long range marksmen. It excels in long-range combat situations and support sniping. It has a laser designator installed, and the scope itself has IFF integrated into it. Uses specialized 10x28 caseless rounds made to work with the guns odd IFF-scope system.  \nIt has an integrated Target Marker and a Laser Targeting system.\n\"Peace Through Superior Firepower\"."
+	name = "\improper T-26 Antimaterial sniper rifle"
```
Comment (hyper2snyper) [inline code/modules/projectiles/guns/specialist.dm:27]:
```suggestion
	name = "\improper T-26 Antimaterial rifle"
```
antimat guns are sniper rifles by definition. this makes the name less long

**Review by hyper2snyper — CHANGES_REQUESTED** [review]

**Review by Kuro020 — CHANGES_REQUESTED** [review]
Give it a new sprite, no reusing existing sprites

**Pull request closed without merging**